test/unit-tests/expression/operators.test.js

```diff
@@ -31,7 +33,7 @@ describe('operators', function () {
   it('should return the precedence of a ParenthesisNode', function () {
     const c = new ConstantNode(1)
 
-    const op = new OperatorNode('or', 'or', [c, c])
+    const op = new OperatorNode('??', 'nullish', [c, c])
 
```

Comment:
Why are you replacing a test that involved logical or with one that involves nullish coalescing? We generally do not want to remove existing tests, and would prefer to only accumulate new tests.